nixos/tests/restic.nix

```diff
@@ -190,6 +197,13 @@ import ./make-test-python.nix (
           'restic-remotebackup snapshots --json | ${pkgs.jq}/bin/jq "length | . == 1"',
 
       )
+
+      # test that the inhibit option is working
+      server.systemctl("start --no-block restic-backups-inhibit-test.service")
+      server.wait_until_succeeds(
+          "systemd-inhibit --no-legend --no-pager | grep -q restic",
+          5
```

Comment:
Wondering if a 5 second timeout is too low for hydra.nixos.org.

I suppose we will see how it goes and can create a follow-up PR if needed :)